apps/src/lib/levelbuilder/lesson-editor/EditTipDialog.jsx

```diff
@@ -55,10 +58,28 @@ export default class EditTipDialog extends Component {
     this.setState({tip: newTip});
   };
 
-  handleClose = () => {
+  handleCloseAndSave = () => {
     this.props.handleConfirm(this.state.tip);
   };
 
+  handleClose = () => {
+    this.props.handleConfirm();
+  };
+
+  handleDelete = () => {
+    this.setState({removeTipOpen: false});
+    this.props.handleDelete(this.state.tip.key);
+    this.props.handleConfirm();
+  };
+
+  handleConfirmDelete = () => {
+    this.setState({removeTipOpen: true});
+  };
```

Comment:
can you please document what these two methods do? I guessed backwards based on the names at first.

alternately there is also the `ConfirmDeleteButton` which you might be able to just drop in here without having to reimplement it, as long as you don't mind the styles.